projects/objects/road/protos/Roundabout.proto

```diff
@@ -150,60 +148,59 @@ PROTO Roundabout [
                 bottom IS bottom
               }
               castShadows IS castShadows
             }
           ]
           name "center"
-          %{ if centerBO then }%
+          %< if (centerBO) { >%
             boundingObject USE CENTER_CYLINDER
-          %{ end }%
+          %< } >%
         }
-      %{ end }%
+      %< } >%
       # for each roads
-      %{ for i = 0, roadNumber - 1 do }%
-        %{
-          local angle         = 2 * math.pi * i / roadNumber + math.pi / roadNumber
-          local x             = outerRadius * math.cos(angle)
-          local z             = outerRadius * math.sin(angle)
-          local rotationAngle = -2 * math.pi * (i + 0.5) / roadNumber + math.pi / 2
-          local theta = math.atan2( roadsWidth/2, outerRadius)
-          local d = roadsWidth/2 - math.sin(theta) * outerRadius
-          local theta2 = math.atan2( roadsWidth/2 + 5 * d, outerRadius)
-          local centralPoint = { x = x, y = z }
-          centralPoint = wbgeometry.findclosest2Dpointinarray(centralPoint, outerPoints)
-          local point = { x = outerRadius * math.cos(angle+theta2), y = outerRadius * math.sin(angle+theta2) }
-          point = wbgeometry.findclosest2Dpointinarray(point, outerPoints)
-          local centralIndex = 0
-          local pointIndex = 0
-          local pointsNumber = wbcore.tablelength(outerPoints)
-          for i = 1, pointsNumber do
-            if wbvector2.equal(outerPoints[i], point) then
-              pointIndex = i
-            elseif wbvector2.equal(outerPoints[i], centralPoint) then
-              centralIndex = i
-            end
-          end
-          numberOfIndex = math.abs(pointIndex - centralIndex)
-        }%
+      %< for (let i = 0; i <= roadNumber - 1; ++i) { >%
+        %<
+          let angle         = 2 * Math.PI * i / roadNumber + Math.PI / roadNumber;
+          let x             = outerRadius * Math.cos(angle);
+          let z             = outerRadius * Math.sin(angle);
+          let rotationAngle = -2 * Math.PI * (i + 0.5) / roadNumber + Math.PI * 0.5;
+          let theta = Math.atan2(roadsWidth * 0.5, outerRadius);
+          let d = roadsWidth * 0.5 - Math.sin(theta) * outerRadius;
+          let theta2 = Math.atan2(roadsWidth * 0.5 + 5 * d, outerRadius);
+          let centralPoint = {x: x, y: z};
+          centralPoint = wbgeometry.findClosestPoint2InArray(centralPoint, outerPoints);
+          let point = {x: outerRadius * Math.cos(angle+theta2), y: outerRadius * Math.sin(angle+theta2)};
+          point = wbgeometry.findClosestPoint2InArray(point, outerPoints);
+          let centralIndex = 0;
+          let pointIndex = 0;
+          let pointsNumber = outerPoints.length;
+          for (let i = 0; i < pointsNumber; ++i) {
+            if (wbvector2.equal(outerPoints[i], point)) {
+              pointIndex = i;
+            } else if (wbvector2.equal(outerPoints[i], centralPoint)) {
+              centralIndex = i;
+            }
+          }
```

Comment:
```suggestion
          for (let i = 0; i < pointsNumber; ++i)
            if (wbvector2.equal(outerPoints[i], point))
              pointIndex = i;
            else if (wbvector2.equal(outerPoints[i], centralPoint))
              centralIndex = i;
```
Cyberbotics rule is to avoid brackets